libr/include/r_io.h

```diff
@@ -474,6 +501,11 @@ R_API bool r_io_addr_is_mapped(RIO *io, ut64 vaddr);
 R_API bool r_io_read_i (RIO* io, ut64 addr, ut64 *val, int size, bool endian);
 R_API bool r_io_write_i (RIO* io, ut64 addr, ut64 *val, int size, bool endian);
 
+#if HAVE_PTRACE
+R_API long r_io_ptrace(RIO *io, r_ptrace_request_t request, pid_t pid, void *addr, r_ptrace_data_t data);
```

Comment:
in linux/ppc the last two args are swapped, i guess this change doesnt affects the behaviour. but just as a random comment

Reply:
Hmm ok, is that documented anywhere? It doesn't really matter here if the types of the args are not swapped.

Reply:
its documented in the manpager of the ppclinux ptrace syscall. but i dont have any linux-ppc now to test so meh :P